supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/UserAction.scala

```diff
@@ -10,7 +10,8 @@ trait UserAction {
       createPipelineParameters.commandScriptContainerPath.pathAsString,
       mounts,
       createPipelineParameters.jobShell,
-      createPipelineParameters.privateDockerKeyAndEncryptedToken
+      createPipelineParameters.privateDockerKeyAndEncryptedToken,
```

Comment:
Please also add this into the `v2beta` backend implementation.

Reply:
Added.